ISISComputingGroup/cs-studio
# 991 Scan Client Receives Commands

Also includes changes from https://github.com/ISISComputingGroup/cs-studio/pull/1

To Test:
* Run the scan server and client
* Confirm that adding/removing a command from the server changes the client
* Confirm that command specific text displayed in the client is the same as it was in the old client
* Confirm that simulating a scan returns correct information from server
* Confirm that all scan features still work (plotting, importing scans and writing/reading from actual IOCs)
